lib/go-tc/deliveryservice_ssl_keys.go

```diff
@@ -93,6 +93,16 @@ type DeliveryServiceSSLKeysV15 struct {
 	Expiration time.Time `json:"expiration,omitempty"`
 }
 
+// SSLKeyExpirationInformation contains information about an SSL key's expiration.
+type SSLKeyExpirationInformation struct {
+	DeliveryService   *string    `json:"deliveryservice"`
+	DeliveryServiceId *int       `json:"deliveryservice_id"`
+	Cdn               *string    `json:"cdn"`
+	Provider          *string    `json:"provider"`
+	Expiration        *time.Time `json:"expiration"`
+	Federated         *bool      `json:"federated"`
```

Comment:
None of these could ever be null, right? Or maybe `DeliveryService` could. But if properties can't be null, then they don't need to be pointers, just causes otherwise unnecessary checking and/or segfaults